.gitattributes

```diff
@@ -0,0 +1,5 @@
+chain/actors/builtin/*/v* linguist-generated=true
+chain/actors/builtin/*/message* linguist-generated=true
+build/openrpc/* linguist-generated=true
```

Comment:
I think the openrpc files, and proxy_gen aren't displayed anyway, so I would drop them.